src/App.js

```diff
@@ -15,12 +16,26 @@ function App() {
       <Nav />
       <Routes>
         <Route path='/' element={<HomePage />} />
-        <Route path='cart' element={<CartPage />} />
         <Route path='login' element={<LoginPage />} />
         <Route path='signup' element={<SignupPage />} />
         <Route path='listing' element={<ProductListings />} />
-        <Route path='wishlist' element={<WishlistPage />} />
         <Route path='mock' element={<Mockman />} />
+        <Route
+          path='cart'
+          element={
+            <RequiresAuth>
+              <CartPage />
+            </RequiresAuth>
+          }
+        />
+        <Route
+          path='wishlist'
+          element={
+            <RequiresAuth>
+              <WishlistPage />
+            </RequiresAuth>
```

Comment:
Instead of writing RequiersAuth multiple time you can make a function 

```
function component(comp) {
            <RequiresAuth>
             {comp}
            </RequiresAuth>
}
```

Reply:
there were only two private routes and I am trying to keep minimum auth files till project submission. Will surely work on this when I implement auth.